commands/util.go

```diff
@@ -65,11 +65,17 @@ func driversForNodeGroup(ctx context.Context, dockerCli command.Cli, ng *store.N
 			return nil, errors.Errorf("failed to find driver %q", f)
 		}
 	} else {
+		// empty driver name is explicit to allow fallbacking to a
```

Comment:
Mention that empty driver means nodegroup was implicitly created as a default driver for a docker context.